docs/content/docs/user-docs/adopted-resource.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Adopting Existing AWS Resources"
+description: "Adopting existing AWS resources that were created using other tools"
+lead: ""
+draft: false
+menu:
+  docs:
+    parent: "getting-started"
+weight: 66
+toc: true
+---
+
+The ACK controllers are intended to manage the complete lifecycle of an AWS
+resource, from creation through until deletion. However, you may already be
+managing those resources using other tools - such as CloudFormation or
+Terraform. Migrating to ACK could be time-consuming to redeclare all resources
+as YAML, or even cause you to lose the state of the application if parts of the
+system are recreated. The ACK `AdoptedResource` custom resource was designed to
+help with migrating the lifecycle of these AWS resources into your Kubernetes
+cluster without having to define the full YAML spec or needing to delete and
+re-create the resource.
+
+To adopt an AWS resource, you need to create an `AdoptedResource` custom
+resource that specifies the unique identifier for the AWS resource and a target
+K8s object. After applying this custom resource to the cluster, the ACK
+controller will describe the AWS resource and create the associated ACK resource
+inside the cluster - with a complete spec and status. The ACK controller will
+then treat the newly-created ACK resource the same way as any other.
+
+All ACK controllers ship with the same `AdoptedResource` CRD. Every controller
+contains the logic for adopting resources from its particular service. That is,
+the S3 controller understands how to adopt all S3 resources. If you don't have a
+particular service controller installed, and try to adopt a resource from that
+service, the `AdoptedResource` CR will have no effect.
+
+## Spec
+
+The full spec for the `AdoptedResource` CRD is located [in the API
+reference][api-ref]. The spec contains two parts: the AWS resource reference and
+the Kubernetes target.
+
+The AWS resource reference requires the unique identifier for the object, either
+as an ARN or as the name or ID of the object. Which of these is required depends
+on the service and the particular resource. You can find which field is required
+by finding the unique identifier field used by the `Describe*` or `List*` API
+calls for that resource.
+
+The Kubernetes target requires the `group` and `kind` - these identify from
+which service and resource you wish to adopt. For example, to adopt an S3
+bucket, you would specify a `group` of `s3.services.k8s.aws` and a `kind` of
+`Bucket`. The Kubernetes target also allows you to override the `metadata` for
+the object that is created. By default, all resources created through an
+`AdoptedResource` will have the same `metadata.name` as the `AdoptedResource`
+that created it. 
+
+[api-ref]: https://aws-controllers-k8s.github.io/community/reference/common/v1alpha1/adoptedresource/
```

Comment:
I think figuring out which field is the required identifier, and what the additional keys are, really needs to be simpler. I would love to put these in the API reference. But since they aren't in the CRDs, we can't auto-generate that bit yet.